docs/src/environment.md

```diff
@@ -23,5 +23,4 @@ MPI.Is_thread_main
 MPI.Initialized
 MPI.Finalize
 MPI.Finalized
-MPI.universe_size
```

Comment:
This is a duplicate of https://github.com/JuliaParallel/MPI.jl/blob/5cf57d8a289a80f7e550a165863ef0dd40667370/docs/src/comm.md#miscellaneous, I feel like this function is better placed in the communication section, but I'm happy to swap it if you think it's better.